src/main.rs

```diff
@@ -52,6 +52,23 @@ fn open_dit_repo() -> Result<Repository> {
 }
 
 
+/// Get a vector of commits from values
+///
+/// This function transforms values to a vector. If no values are supplied,
+/// an empty vector will be returned.
+///
+fn values_to_hashes<'repo>(repo: &'repo Repository, values: Option<Values>) -> Result<Vec<Commit<'repo>>> {
```

Comment:
It was intended to reduce boiler plate. For some reason the clap guys chose to return `None` instead of just an empty `Values` in cases where the option was not supplied.